src/main.build.js

```diff
@@ -3,7 +3,7 @@ import Graph2d from './components/Graph2d.vue';
 import Network from './components/Network.vue';
 
 export {
-    Timeline,
-    Graph2d,
-    Network
-};
+	Timeline,
+	Graph2d,
+	Network
+};
```

Comment:
Please fix the no new line error

Reply:
This one I didn't get... What you mean by "no new line error"?

Reply:
It's just poping in the git review. You should just add a new line at the end of the js file.
Here is why https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline